### PR TITLE
Updated scheduler portions of the documentation

### PR DESCRIPTION
**Pull request by cppwfs**
Resolves https://github.com/spring-cloud/spring-cloud-dataflow/issues/3642

### CODE REVIEW

File: spring-cloud-dataflow-docs/src/main/asciidoc/tasks.adoc [hunk 1]
```diff
@@ -1105,6 +1105,9 @@ A schedule can be created through the RESTful API or the Spring Cloud Data Flow
 Spring Cloud Data Flow will schedule the execution of its tasks via a scheduling agent that is available on the cloud platform.
 When using the Cloud Foundry platform Spring Cloud Data Flow will use the https://www.cloudfoundry.org/the-foundry/scheduler/[PCF Scheduler].
 When using Kubernetes, a https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/[CronJob] will be used.
+The way that Spring Cloud Data Flow does this is by deploying a SchedulerTaskLauncher application to the platform and then
+the scheduling agent is notified to schedule the SchedulerTaskLauncher.  When the scheduling agent launches the
```
Comment (sabbyanandan) [inline spring-cloud-dataflow-docs/src/main/asciidoc/tasks.adoc]: "scheduling agent" <- sounds generic; maybe add in parens: "(e.g., CronJob in Kubernetes)"

File: spring-cloud-dataflow-docs/src/main/asciidoc/tasks.adoc [hunk 2]
```diff
@@ -1131,7 +1135,7 @@ You can schedule a task execution via the:
 * RESTful API
 * Spring Cloud Data Flow Dashboard
 
-To schedule a task from the UI click the Tasks tab at the top of the screen, this will take you to the Task Definitions screen.   Then from the Task Definition that you wish to schedule click the "clock" icon associated with task definition you wish to schedule.
+To schedule a task from the UI click the Tasks tab at the top of the screen, this will take you to the Task Definitions screen.   From the Task Definition that you wish to schedule click the drop down icon and then select the `Schedule Task` option.
```
Comment (sabbyanandan) [inline spring-cloud-dataflow-docs/src/main/asciidoc/tasks.adoc]: Maybe:
> From the Task Definition that you wish to schedule, please click the dropdown icon and then select the `Schedule Task` option.

File: spring-cloud-dataflow-docs/src/main/asciidoc/tasks.adoc [hunk 3]
```diff
@@ -1143,6 +1147,16 @@ You can delete a schedule via the:
...
 * Spring Cloud Data Flow Dashboard
 
-To delete a schedule through the dashboard, select the Schedule tab under Tasks tab and click the `garbage can` icon next to the schedule you wish to delete.
+To delete a schedule through the dashboard, select the Schedule tab under Tasks tab and click the dropdown icon next to the schedule you wish to delete and select the `Delete schedule` option.
 
 NOTE: Any currently running tasks that were run by the scheduling agent will not be stopped if the schedule is deleted.   It only prevents future executions.
+
+[[spring-cloud-data-flow-schedule-list]]
+==== Listing Schedules
+To view your tasks click the Tasks tab and then click the Schedules tab on the Tasks page.
```
Comment (sabbyanandan) [inline spring-cloud-dataflow-docs/src/main/asciidoc/tasks.adoc]: > To view your schedules for a given task, ....

File: spring-cloud-dataflow-docs/src/main/asciidoc/tasks.adoc (hunk 1 above)
Comment (sabbyanandan) [reply spring-cloud-dataflow-docs/src/main/asciidoc/tasks.adoc]: For "SchedulerTaskLauncher", do we want to link to the GH module?

Comment (cppwfs) [reply spring-cloud-dataflow-docs/src/main/asciidoc/tasks.adoc]: Updated.  Thank you!

**Comment by sabbyanandan** [comment]
Let's go! 👍

**Comment by cppwfs** [comment]
Squashed Merged

**Pull request closed without merging**